.github/workflows/test-server-self-host.yml

```
@@ -0,0 +1,139 @@
+          docker logs searxng > searxng/searxng.log 2>&1
+          docker kill searxng
+        working-directory: ./
+      - uses: actions/upload-artifact@v4
+        if: always()
+        with:
+          name: Logs (${{ matrix.ai }}, ${{ matrix.search }}, ${{ matrix.engine }}, ${{ matrix.proxy }})
+          path: |
+            ./apps/api/api.log
+            ./apps/api/worker.log
+      - uses: actions/upload-artifact@v4
+        if: always() && matrix.playwright
```

actionlint (1.7.4)

property "playwright" is not defined in object type {ai: string; engine: string; proxy: string; search: string}

(expression)